WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingView.swift

```diff
@@ -0,0 +1,211 @@
+            .toolbar {
+                ToolbarItem(placement: .cancellationAction) {
+                    Button(Localization.cancel) {
+                        dismiss()
+                    }
+                }
+            }
+        }
+    }
+    private func sectionHeading(title: String) -> some View {
+        Text(title.uppercased())
+            .subheadlineStyle()
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .padding(Layout.sectionHeadingPadding)
+    }
+
+    private func destinationItem(title: String,
+                                 subtitle: String,
+                                 type: BlazeAdDestinationSettingViewModel.DestinationURLType,
+                                 showBottomDivider: Bool = false) -> some View {
+        HStack(alignment: .center) {
+            if type == viewModel.selectedDestinationType {
+                Image(systemName: "checkmark")
+                    .padding(.leading, Layout.contentSpacing)
+                    .padding(.trailing, Layout.contentHorizontalSpacing)
+                    .foregroundColor(Color(uiColor: .accent))
+            } else {
+                Image(systemName: "checkmark")
+                    .padding(.leading, Layout.contentSpacing)
+                    .padding(.trailing, Layout.contentHorizontalSpacing)
+                    .hidden() // Small hack to make the icon space consistent while not showing the icon.
+            }
+
+            VStack(alignment: .leading) {
+                Text(title)
+                    .bodyStyle()
+                    .padding(.top, Layout.contentVerticalSpacing)
+                Text(subtitle)
+                    .subheadlineStyle()
+                    .multilineTextAlignment(.leading)
+                    .padding(.bottom, Layout.contentVerticalSpacing)
+
+                if showBottomDivider {
+                    Divider()
+                        .background(Color(.systemGray3))
+                }
+            }
+        }
+        .frame(maxWidth: .infinity, alignment: .leading)
+        .background(Color(.systemBackground))
+        .onTapGesture {
+            viewModel.setDestinationType(type: type)
+        }
+    }
+
+    private func parameterItem(itemName: String) -> some View {
+        VStack {
+            HStack {
+                Text(itemName)
+                    .padding([.top, .bottom], Layout.parametersVerticalSpacing)
+                Spacer()
+                Image(systemName: "chevron.right")
+                    .foregroundColor(Color(.systemGray4))
+                    .padding(.leading, 8)
+                    .padding(.trailing, Layout.contentSpacing)
+            }
+            .padding(.leading, Layout.contentSpacing)
+            .frame(maxWidth: .infinity, alignment: .leading)
+
+            Rectangle()
+                .frame(height: 1)
+                .foregroundColor(Color(.systemGray3))
+                .padding(.leading, Layout.contentSpacing)
```

Comment:
A custom Divider due to the need to have some leading spacing like in the design.

Reply:
Will be deleted anyway soon a I'm moving things to be a List so it can be swiped to delete so feel free to ignore this area for now.

Reply:
AFAIK you can still add padding to the default Divider.

Reply:
This area is modified already in (turned ito a list)

https://github.com/woocommerce/woocommerce-ios/pull/11642/commits/397c70d355cf355bee35547646d17ab604fceb1e

Reply:
Please turn the whole screen to a List because otherwise, you'll have a sticky header and only the parameter list is scrollable.

Reply:
Yeah, that's what I've been discovering, it's weird to see the parameters scrolling alone. I'm slowly learning more about List behavior as we speak. 

I'll get that in the next PR.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/11642/commits/a281f6017ddb817382ceb168f344023a77f60c91